pipeline/config-prod.groovy

```diff
@@ -301,6 +305,8 @@ environments {
                     cpu_limit = "400m"
                     memory_request = "1.5Gi"
                     memory_limit = "3Gi"
+                    uwsgi_threads = 2
+                    uwsgi_processes = 4
                     replica_min = 2
                     replica_max = 4
```

Comment:
how do we feel about more smaller pods? like 3-6(0r 8) instead of 2-4? with less resources each. Celcio made a good point that we can't really say HA unless we have 3, cause we might lose one pod at any given time.

Reply:
That's a good idea! something we can move towards possibly 🤷‍♂ Definitely has its pros/cons and needs some manual testing. I would argue that having 2 pods still makes you HA just as much as 3 would but im sure he had a good argument as well.
Also all our deploys and such will take longer the more pods we throw on there 🤔 

I didnt alter any test/prod resources in this PR just to keep the scope limited but maybe we can look into that more?